engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -396,6 +423,7 @@ public void setExitStatus(int exitStatus)
     public ChangeLogicalPlanEvent(LogicalPlanRequest request)
     {
       this.request = request;
+      this.setEventLogLevel(LogLevel.INFO);
```

Comment:
if stopContainerEvent is at WARN level shouldn't startContainerEvent be also at WARN level?